data/scenarios/Testing/378-objectives.yaml

```diff
@@ -0,0 +1,64 @@
+name: Complete two objectives in succession.
+description: |
+  First, get some trees; then, use them to build a harvester.
+objectives:
+  - goal:
+      - Your first goal is to get three trees.
+    condition: |
+      try {
+        n <- as base {count "tree"};
+        return (n >= 3)
+      } { return false }
+  - goal:
+      - Nice job.  Now, build a harvester.
+    condition: |
+      try { as base {has "harvester"} } {return false}
+solution: |
+  build {turn right; move; move; grab; move; grab; move; grab; turn back; move; move; move; move};
+  wait 16;
+  salvage;
+  make "log"; make "log"; make "log";
+  make "board"; make "board"; make "board";
+  make "box";
+  make "wooden gear"; make "wooden gear";
+  make "harvester"
+robots:
+  - name: base
+    display:
+      char: 'Ω'
+      attr: robot
+    dir: [0,1]
+    devices:
+      - 3D printer
+      - dictionary
+      - grabber
+      - life support system
+      - logger
+      - toolkit
+      - solar panel
+      - workbench
+      - clock
+    inventory:
+      - [5, 3D printer]
+      - [100, treads]
+      - [70, grabber]
+      - [100, solar panel]
+      - [50, scanner]
+      - [5, toolkit]
```

Comment:
I prefer keeping the numbers the same, like 10, to make it easier to see what is important.
Also, `[0, harvester]` would be nice.

But for testing it of course does not matter much.